charts/k8ssandra/templates/stargate/stargate.yaml

```diff
@@ -111,11 +111,13 @@ spec:
               path: /checker/liveness
               port: health
             timeoutSeconds: 10
-            initialDelaySeconds: 240
+            initialDelaySeconds: 30
```

Comment:
These sound good for defaults, but we should make the probes configurable.

Reply:
Good idea, implemented in eac44a5.